amy/emails/forms.py

```diff
@@ -1,10 +1,48 @@
 from django import forms
+from markdownx.fields import MarkdownxFormField
 
-from emails.models import ScheduledEmail
+from emails.models import EmailTemplate, ScheduledEmail
 from workshops.forms import BootstrapHelper
 
 
+class EmailTemplateCreateForm(forms.ModelForm):
+    body = MarkdownxFormField(
+        label=EmailTemplate._meta.get_field("body").verbose_name,
+        help_text=EmailTemplate._meta.get_field("body").help_text,
+        widget=forms.Textarea,
+    )
+
+    class Meta:
+        model = EmailTemplate
+        fields = [
+            "name",
+            "signal",
+            "from_header",
+            "reply_to_header",
+            "cc_header",
+            "bcc_header",
+            "subject",
+            "body",
```

Comment:
@elichad restrictions for `signal` are coming, for example here I linked signals with their string names: https://github.com/carpentries/amy/pull/2481/commits/3bd036cfadda7cb83c6ba70e047956eb8bb8ac81. As you noted, it will be further limited to the user, either through read-only field or (in combination with) drop-down selection.

The context documentation is a really good idea and I put a ticket for it: https://github.com/carpentries/amy/issues/2482